cmd/contour/gatewayprovisioner.go

```diff
@@ -41,25 +42,29 @@ func registerGatewayProvisioner(app *kingpin.Application) (*kingpin.CmdClause, *
 	}
 
 	cmd.Flag("contour-image", "The container image used for the managed Contour.").
-		Default(config.contourImage).
-		StringVar(&config.contourImage)
+		Default(conf.contourImage).
+		StringVar(&conf.contourImage)
 
 	cmd.Flag("envoy-image", "The container image used for the managed Envoy.").
-		Default(config.envoyImage).
-		StringVar(&config.envoyImage)
+		Default(conf.envoyImage).
+		StringVar(&conf.envoyImage)
 
 	cmd.Flag("metrics-addr", "The address the metric endpoint binds to. It can be set to 0 to disable serving metrics.").
-		Default(config.metricsBindAddress).
-		StringVar(&config.metricsBindAddress)
+		Default(conf.metricsBindAddress).
+		StringVar(&conf.metricsBindAddress)
+
+	cmd.Flag("leader-election-namespace", "The namespace in which the leader election resource will be created.").
+		Default(config.GetenvOr("CONTOUR_NAMESPACE", "projectcontour")).
```

Comment:
we don't currently set the namespace in the example manifest for the gateway provision, so i think this can be simplified a bit
```suggestion
		Default("projectcontour").
```